pkg/scheduler/event_handler.go

```diff
@@ -225,6 +288,9 @@ func (s *Scheduler) deleteCluster(obj interface{}) {
 
 	klog.V(3).Infof("Delete event for cluster %s", cluster.Name)
 
+	s.enqueueAffectedPolicy(cluster)
+	s.enqueueAffectedClusterPolicy(cluster)
```

Comment:
Should we also be unaware of cluster deletion?

Reply:
+1